src/main/java/com/extratime/shops/controller/ShopController.java

```diff
@@ -0,0 +1,19 @@
+package com.extratime.shops.controller;
+
+import com.extratime.shops.entity.Shop;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.concurrent.atomic.AtomicLong;
+
+@RestController
+public class ShopController {
+
+    private final AtomicLong counter = new AtomicLong();
+
+    @RequestMapping("/shop")
```

Comment:
Please, change from `@RequestMapping` to `@GetMapping` and the Rest resources should be in the plural.